components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/endpoint/registry/util/EndpointRegistryUtil.java

```diff
@@ -32,4 +46,24 @@ public static void raiseResourceAlreadyExistsException(String msg) throws
         throw new EndpointRegistryResourceAlreadyExistsException(msg);
     }
 
+    public static boolean isValidDefinition(URL definitionURL, String definitionType)
+            throws EndpointRegistryException {
```

Comment:
Define a different exception for this

Reply:
Added in https://github.com/wso2/carbon-apimgt/pull/8611/commits/c4274550288178f609363deee99b5cb631915ed2